src/app/api/send/route.ts

```
@@ -0,0 +1,40 @@
+import { NextResponse } from 'next/server';
+import { Resend } from 'resend';
+import prisma from '@/utils/prisma';
+
+export async function POST(request: Request) {
+  try {
+    const { email, name } = await request.json();
+
+    const resend = new Resend(process.env.RESEND_API_KEY);
+
+    const contact = resend.contacts.create({
+      email: email,
+      firstName: name,
+      unsubscribed: false,
+      audienceId: '274752ed-c875-4c06-aa9e-984d5b96df11',
+    });
+
+    const userAccountId = (await contact).data?.id;
```

Instead of this - please add `await` before `resend.contacts.create`
Also, if it wasn't created - let's return an error